randovania/games/samus_returns/logic_database/Area 1.txt

```diff
@@ -916,11 +916,16 @@ Extra - asset_id: collision_camera_035
           All of the following:
               Phase Drift
               Any of the following:
+                  Climb Rooms Vertically (High Jump)
```

Comment:
This seems redundant since `Climb Rooms Vertically` is already used without Phase Drift but no High Jump. Change this to be High Jump instead.